### DISCUSSION TITLE
Suggestions: Update for Windows 10 20H2 and Windows 11

### DISCUSSION THREAD

**Discussion by kiquenet** [farag2/Sophia-Script-for-Windows#442] [General]
Is your feature request related to a problem?.

Only improvements like:

Unpin to taskbar Microsoft Store, Mail, ... apps
Pin to taskbar any apps like Remote Desktop

Set SearchProvider and SearchProviderUrl for Google in Edge navigator.

Pin to QuickAccess

Pin to Start Menu

Connect to network drive

Shortcuts

Describe the solution you'd like

Pin / Unpin to taskbar
https://stackoverflow.com/questions/9739772/how-to-pin-to-taskbar-using-powershell

Hidden Cortana button

Edge
https://stackoverflow.com/questions/60892120/edge-chromium-gp-how-to-set-google-as-default-bypass-first-run-dialog-and-ha

Pin to QuickAccess
https://stackoverflow.com/questions/30051634/is-it-possible-programmatically-add-folders-to-the-windows-10-quick-access-panel

Pin to Start Menu
https://stackoverflow.com/questions/9903847/how-to-pin-to-start-menu-using-powershell

Connect to network drive
https://www.itechguides.com/how-to-map-a-drive-in-powershell/

Create shortcut
https://stackoverflow.com/questions/9701840/how-to-create-a-shortcut-using-powershell

Describe alternatives you've considered
Using powershell

Additional context
Windows 10 20H2

**Comment by farag2** [comment]
No. I use my own method. That previous one is absolete and totally outdated (doesn't work now). Actually what's your issue?

> Windows 10 20H2

Outdated and unsupported by MS Windows

Also Windows 10 uses different method to pin/unpin tiles unlike Windows 11

> Windows 11

I can only unpin all tiles at once due to now the Start scheme is a obfuscated JSON file.
